src/app/cli/src/app.rs

```diff
@@ -85,7 +85,7 @@ pub async fn run(
         .transpose()
         .map_err(CLIError::usage_error_from)?
         .map(Into::into);
-    let is_e2e_testing = matches.get_flag("e2e-testing");
+    let is_e2e_testing = matches.get_one::<PathBuf>("e2e-output-data-path").is_some();
```

Comment:
Such as we don't care about value we can use `get_raw` method to avoid passing type here

Reply:
a good find -- really, we don't even need a type

used `contains_id()`, which is just fine for this (checking for availability)